state-chain/pallets/cf-threshold-signature/src/lib.rs

```diff
@@ -709,6 +721,15 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 					request_id,
 					attempt_count
 				);
+
+				T::CfeEventEmitter::signature_request(
+					ceremony_id,
+					epoch,
+					key.clone(),
+					participants.clone(),
+					payload.clone(),
+				);
+
 				Event::<T, I>::ThresholdSignatureRequest {
```

Comment:
So the idea will be to leave these events here so that the upgrade is made a little simpler? Then once everyone's upgrading, we can remove the emission of these events?

Reply:
Yeah, I was going to ask whether we want to keep them or not. I prefer if we delete them, but would need to make sure that no one else is using these.

Reply:
I think one strong reason to leave this at the moment is all the alerting infrastructure we have. I'm pretty sure some of the alerts use these events, we should convert them over before removing them. @marcellorigotti fyi

Reply:
I created an issue: https://linear.app/chainflip/issue/PRO-1096/move-alerts-that-depend-on-events-to-use-the-new-cfe-event-emitter

Reply:
thanks! will talk to tom and have a look to be sure that we are not using these/to change these alerts

Reply:
@marcellorigotti checked and it seems we don't use any of these moved events in the alerts - so in that case I think we can remove these. But might be worth doing in a separate PR just in case it needs to be rolled back or something.